src/js/DrawController.js

```diff
@@ -45,7 +45,7 @@ goog.provide('ga_draw_controller');
              left: 60
           },
           height: 145,
-          elevationModel: 'COMB'
+          elevationModel: gaGlobalOptions.defaultElevationModel
```

Comment:
If we set the `defaultElevationModel` at line 59 we don't need to set the elevation model here.